src/libcore/atomic.rs

```diff
@@ -1089,3 +1090,21 @@ pub fn fence(order: Ordering) {
         }
     }
 }
+
+macro_rules! impl_Debug {
+    ($($t:ident)*) => ($(
+        impl fmt::Debug for $t {
```

Comment:
Other impls are still marked, though. Even if it's insta-stable having the version information of *when* an impl appeared can be useful for understanding the version requirements of your code.